content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -54,6 +54,12 @@ Code Hotspots identification is enabled by default when you [turn on profiling f
 
 Requires `dd-trace-py` version 0.44.0+.
 
+To enable the new [timeline feature](#span-execution-timeline-view)
+
+- Upgrade `dd-trace-py` to version 2.10.4 or later
+- Set environment variable `DD_PROFILING_TIMELINE_ENABLED` to `true`
+- Set environment variable `DD_PROFILING_EXPORT_LIBDD_ENABLED` to `true`, this enables new profiling exporter implemented as a native extension.
```

Comment:
```suggestion
- Set environment variable `DD_PROFILING_EXPORT_LIBDD_ENABLED` to `true`. This enables the profiling exporter to be implemented as a native extension.
```

Trying to fix grammar—does my change preserve the intended meaning of this sentence?

Reply:
This bit is no longer needed.